### ISSUE TITLE
TLS connection example doc

### ISSUE THREAD

**Issue by srikang85** [vertica/vertica-nodejs#107]
Could you please add some examples of client code when the Vertica server has TLS enabled

**Comment by hulmgulm-at-work** [comment]
I tried to open a TLS connection with the following connection options:
![image](https://user-images.githubusercontent.com/19646718/208666259-4a72323f-dc9f-4ed0-8f4f-ac6a06f68fbd.png)

The connection fails with this strange error message:

```
/home/hulmi/tmp/node_modules/v-protocol/dist/parser.js:344
        const message = name === 'notice' ? new messages_1.NoticeMessage(length, messageValue) : new messages_1.DatabaseError(messageValue, length, name);
                                                                                                 ^

error: Invalid startup packet layout: expected terminator as last byte
    at Parser.parseErrorMessage (/home/hulmi/tmp/node_modules/v-protocol/dist/parser.js:344:98)
    at Parser.handlePacket (/home/hulmi/tmp/node_modules/v-protocol/dist/parser.js:144:29)
```

Digging into the package received by vertica-nodejs I got the following:
`E�SFATALC08V01V3753MInvalid startup packet layout: expected terminator as last byteF/data/qb_workspaces/jenkins2/ReleaseBuilds/Hammermill/REL-10_1_1-x_hammermill/build/vertica/Session/ClientSession.cppL4865RProcessStartupPacket`

Any idea how to get TLS to work?

**Comment by DMickens** [comment]
Will be giving this some priority shortly. Apologies for the holiday delay

**Comment by DMickens** [comment]
"expected null terminator" problem is an issue resolved by https://github.com/vertica/vertica-nodejs/pull/115. 

Functionally TLS should be working fine. Will add documentation

**Comment by srikang85** [comment]
> "expected null terminator" problem is an issue resolved by #115.
> 
> Functionally TLS should be working fine. Will add documentation

Thank you @DMickens for fixing the issue. Is this fix not published yet?

**Comment by DMickens** [comment]
> > "expected null terminator" problem is an issue resolved by #115.
> > Functionally TLS should be working fine. Will add documentation
> 
> Thank you @DMickens for fixing the issue. Is this fix not published yet?

Not yet, planning on publishing on 2/6 with updated documentation
